src/terminal/parser/ut_parser/OutputEngineTest.cpp

```diff
@@ -173,12 +173,7 @@ class Microsoft::Console::VirtualTerminal::OutputEngineTest final
             Log::Comment(L"Escape from DcsPassThrough");
             shouldEscapeOut = false;
             mach._state = StateMachine::VTStates::DcsPassThrough;
-            break;
-        }
-        case 17:
-        {
-            Log::Comment(L"Escape from DcsTermination");
-            mach._state = StateMachine::VTStates::DcsTermination;
+            mach._dcsStringHandler = [](const auto) { return true; };
             break;
         }
         case 18:
```

Comment:
I'm concerned about this -- it looks like we're going from case 16 to 18, but the test data only runs to 17. :smile:

Reply:
Oops. That was a dumb mistake. Thanks for catching that.